.github/labeler.yml

```diff
@@ -1,6 +1,9 @@
-# Make sure any dependency changes are benchmarked (only changes to the locks
-#  make a material difference - changes to the Conda YAML files are not
-#  benchmarked).
+# benchmark_this automatically triggers the benchmark workflow when added by
+#  a user. No triggering happens when GitHub Actions adds the label (this
+#  avoids security vulnerabilities), so alternative triggers for the below
+#  files are therefore included in workflows/benchmarks_run.yml. Automatic
+#  labelling is still included here to make it easier to search pull requests,
+#  and to reinforce the culture of using this label.
 benchmark_this:
 - changed-files:
-  - any-glob-to-any-file: 'requirements/locks/*.lock'
+  - any-glob-to-any-file: ['requirements/locks/*.lock', "setup.py", "pyproject.toml"]
```

Comment:
After conversation with @trexfeathers, we came to the decision to remove the pyproject.toml from this list, and in the benchmark action as there is much more that can be changed within the pyproject.toml that do not affect the benchmarks.